src/database.cpp

```diff
@@ -4705,7 +4705,7 @@ QList<TableParams> Database::makeTableParams()
       "lauter_deadspace" << "top_up_kettle" << "hop_utilization" <<
       "notes";
    tmp.newElement =
-      (NewIngFunc) (Equipment*(Database::*)(void))
+      (NewIngFunc) (Equipment*(Database::*)())
```

Comment:
Leave the `(NewIngFunc)` cast and remove the `(Equipment*(Database::*)())` cast.